restApi/src/main/java/bisq/restApi/dto/DtoUtil.java

```diff
@@ -0,0 +1,45 @@
+package bisq.restApi.dto;
+
+import java.beans.BeanInfo;
+import java.beans.Introspector;
+import java.beans.PropertyDescriptor;
+import java.lang.reflect.Method;
+
+public class DtoUtil {
+
+    public static <T> T copyBeanProperties(Object fromObject, T toObject) {
```

Comment:
I am not a fan of interfaced where arguments get altered, is against functional programming rules.
Also we return the same value we pass... 
As the toObject is used as class only, we could pass the class and also construct the object (i think it is always used with no args) here. Then I think a static context from the caller would also work better.

Reply:
yep, was my first thought as well. Basically its a tradeoff, use static create methods, then we need to have it always,even if copyBeanProperties for simple Dto would be enough. Because the static creation method cant be inherited. Which way shall we go?